### PR TITLE
V1.0.0

### PR DESCRIPTION
**Pull request by erlendvollset**

### CODE REVIEW

**Comment by f1cognite** [comment]
Holy... what a PR 😃 
I need some time to review it & test it.

I guess that this is WIP @erlendvollset ?

File: cognite/client/api/datapoints.py [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+
+    def insert_multiple(self, datapoints: List[Dict[str, Union[str, int, List]]]):
+        valid_dps_objects = []
+        for dps_object in datapoints:
+            for key in dps_object:
+                if key not in ("id", "externalId", "datapoints"):
+                    raise AssertionError(
+                        "Invalid key '{}' in datapoints. Must contain 'datapoints', and 'id' or 'externalId".format(key)
+                    )
+            valid_dps_object = dps_object.copy()
```
Comment (NilsBarlaug-cognite) [inline cognite/client/api/datapoints.py]: You said it's unnecessary

File: cognite/client/api/datapoints.py [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+            "aggregates": aggregates,
+            "granularity": granularity,
+            "includeOutsidePoints": include_outside_points,
+            "limit": self._LIMIT_AGG if aggregates else self._LIMIT,
+        }
+        res = self._post(self._RESOURCE_PATH + "/get", json=payload).json()["data"]["items"][0]
+        return Datapoints._load(res)
+
+    @staticmethod
+    def _concatenate_datapoints(*dps_objects: Datapoints) -> Datapoints:
```
Comment (NilsBarlaug-cognite) [inline cognite/client/api/datapoints.py]: More confusing than value-adding with `*` in this case?

**Comment by erlendvollset** [comment]
> Holy... what a PR 😃
> I need some time to review it & test it.
> 
> I guess that this is WIP @erlendvollset ?

@f1cognite 
It's not as bad as it looks most of the 36k lines is openapi spec stuff.
But yes, WIP. I have moved the current master branch to the `0.13` branch, so I figured we could merge this into master now and I will make (much) smaller PRs against that going forward.

File: cognite/client/api/datapoints.py [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+        assert 1 == len(set([dps.id for dps in dps_objects]))
+        assert 1 == len(set([dps.external_id for dps in dps_objects]))
+        concat_dps_object = Datapoints(id=dps_objects[0].id, external_id=dps_objects[0].external_id)
+        for dps in dps_objects:
```
Comment (NilsBarlaug-cognite) [inline cognite/client/api/datapoints.py]:
```suggestion
        for dps in dps_objects[1:]:
```
?

**Pull request merged**